src/packages/@ncigdc/utils/filtersToName.js

```diff
@@ -19,14 +19,14 @@ function getValues(filters, sets) {
 }
 
 const MAX_VALUES = 6;
-export default function({ filters, max = MAX_VALUES, sets }) {
+export default function({ filters, max = MAX_VALUES, sets, length = 100 }) {
```

Comment:
should this `length = 100` be `length =  MAX_SET_NAME_LENGTH` or is that a different 100

Reply:
100 here was just a default, it can be removed and I can make it unlimited if it's not passed in.